operator/internal/manifests/internal/config/options.go

```diff
@@ -68,6 +68,14 @@ type GossipRing struct {
 	BindPort int
 	// MembersDiscoveryAddr is required
 	MembersDiscoveryAddr string
+	// Zone is optional
+	Zone *ZoneConfig
+}
+
+// ZoneConfig  defines the zone aware configuration for Loki
+type ZoneConfig struct {
+	// InstanceAvailabilityZone is required, informs loki where this instance lives
+	InstanceAvailabilityZone string
```

Comment:
I suggest to skip adding an extra struct and keep this simple with a field `InstanceAvailabilityZone string` in in the `GossipRing` struct.